### PR TITLE
Added Topic list component

### PR DESCRIPTION
**Pull request by Silent777**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/secure/RapidBoard.jspa?rapidView=738&view=detail&selectedIssue=LVPYTP-56)

## Code reviewers

- [ ] @dmytrorevak
- [ ] @IvanAndrukh
- [ ] @JjuliaP
- [ ] @makarman
- [ ] @noobhunters
- [ ] @Silent777
- [ ] @Staryj812

### Second Level Review

- [ ] @ssOleg
- [ ] @lhalam

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: eventually/static/src/containers/topicList/AssignmentLink.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+export default class AssignmentItem extends React.Component {
+    
+        constructor(props) {
+            super(props);
+            this.state = {
+                expanded: false,
+            };
+        }
+render(){
```
Comment (redmyt) [inline eventually/static/src/containers/topicList/AssignmentLink.js:18]: `render() {` not `render(){`

File: eventually/static/src/containers/topicList/AssignmentList.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+    constructor(props) {
+        super(props);
+    }
+
+    render() {
+        return (
+            <div style={style}>
+                    {
+                        assignmentsListService().assignments.map(assignment => (
```
Comment (redmyt) [inline eventually/static/src/containers/topicList/AssignmentList.js:22]: invocation of `assignmentsListService` shouldn't be here

File: eventually/static/src/containers/topicList/TopicItem.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import {Card, CardActions, CardHeader, CardText} from 'material-ui/Card';
+import FlatButton from 'material-ui/FlatButton';
+import AssignmentList from './AssignmentList';
+
+
+const style = {
+    color: '#455A64',
```
Comment (redmyt) [inline eventually/static/src/containers/topicList/TopicItem.js:8]: you can use colors of material-ui. there is the cool table with it.

File: eventually/static/src/containers/topicList/TopicItem.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    constructor(props) {
+        super(props);
+        this.state = {
+            expanded: this.props.isActive,
+        };
+    }
+
+    cangeExp = (newExpandedState) => {
+        this.props.change(this.props.id);
+        // this.setState({expanded: newExpandedState});
```
Comment (redmyt) [inline eventually/static/src/containers/topicList/TopicItem.js]: forget =)

**Review by redmyt — DISMISSED** [review]

File: eventually/static/src/containers/topicList/TopicItem.js (hunk 3 above)
Comment (Silent777) [reply eventually/static/src/containers/topicList/TopicItem.js:8]: Hello dear friend )

**Review by redmyt — APPROVED** [review]

**Pull request merged**